strapi/strapi
# feat(blocks): Ability to expand blocks input

### What does it do?

Adds the ability to expand blocks input

### How to test it?

On clicking expand button at the bottom-right of blocks input, editor should open in fullscreen with content aligned in the middle of the modal as per [design](https://www.figma.com/file/MWPKHNT98LLWHGRp3MyWqu/Blocks?type=design&node-id=1333-5165&mode=design&t=Lu6K0vRvzYk99OKD-0).
